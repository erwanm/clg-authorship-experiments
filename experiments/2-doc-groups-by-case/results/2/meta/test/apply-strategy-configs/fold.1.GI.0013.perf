0.437976	0.663600	0.660000
