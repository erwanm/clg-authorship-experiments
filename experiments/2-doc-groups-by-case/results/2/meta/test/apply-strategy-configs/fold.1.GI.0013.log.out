apply-config.sh: applying config '/tmp/expe2/2/process/selected-meta-configs/fold.1.0062.model/strategy-configs/fold.1.GI.0013' to cases './results/2/meta/test/apply-strategy-configs/only-cases.tmp.mL4g8zxZ'; results written to './results/2/meta/test/apply-strategy-configs/fold.1.GI.0013'
apply-config.sh: done.
